lost/src/main/java/com/mapzen/android/lost/internal/FusionEngine.java

```diff
@@ -112,6 +118,10 @@ public FusionEngine(Context context, Callback callback) {
     if (passiveInterval < Long.MAX_VALUE) {
       enablePassive(passiveInterval);
     }
+
+    if (prepLooper) {
+      Looper.loop();
```

Comment:
>the app is killed by the user
If we are talking force stop here, then on the Android 3.1+ the app won't receive any intents after that.
>FusionEngine on the other hand runs in a local service
As far as I can see there is a LocationManager under the hood. Maybe you should simply wrap up existing calls like this [void requestLocationUpdates (long minTime, 
                float minDistance, 
                Criteria criteria, 
                PendingIntent intent)](https://goo.gl/W40jYS)?